digits/templates/datasets/images/classification/show.html

```diff
@@ -101,6 +101,17 @@
     <b>Image Mean:</b>
     <img src="{{url_for('serve_file', path=task.path('mean.jpg', relative=True))}}" />
     {% endif %}
+    <br/>
+    <b>Augmentations:</b>
```

Comment:
This should only be shown if some augmentations exist. Otherwise you end up with this:

![augmentations-none-display-issue](https://cloud.githubusercontent.com/assets/687269/10109099/c7aabbea-6379-11e5-9042-d44d16a7e798.jpg)